### PR TITLE
add login

### PR DESCRIPTION
**Pull request by PouriyaHos**
login added

### CODE REVIEW

File: Login/login.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+window.onload = ()=> {
+    this.sessionStorage.setItem("username", "api-client");
+    this.sessionStorage.setItem("password", "KEV9EwC5SEvk4dF");
```
Comment (josipgrgic) [inline Login/login.js:4]: We don't initialize the session storage with any values. As I said, adding login information to GitHub is a big security risk! 
Please delete these lines

File: Login/login.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+var login = document.getElementById('log-in');
+var form = document.querySelector('form');
+form.onsubmit = ()=>{return false}
+
+login.onclick = ()=>{
+
+    if ((input[0].value != "") && (input[1].value != ""))
+    {
+        if ((input[0].value == sessionStorage.getItem("username")) && (input[1].value 
+            == sessionStorage.getItem("password")))
```
Comment (josipgrgic) [inline Login/login.js:21]: This code is too complicated.

The flow should look like this:
- get the values from the input elements
- if the username input value **is empty** **or** the password input value **is empty** 
  - then add an error
  - return
- if the username input value **and** the password input value **are not empty**
  - store the username and password in the session storage
  - return


And that's it

Comment (josipgrgic) [reply Login/login.js:21]: So you don't need to compare usernames and passwords

**Review by josipgrgic — CHANGES_REQUESTED** [review]
Please resolve the issues I pointed out :)

**Comment by josipgrgic** [comment]
You also need to add a link to the login page from the home page so that users know it exits

**Review by josipgrgic — CHANGES_REQUESTED** [review]
Did you notice the strange characters in your code? These appeared because your branch has merge conflicts with the main branch. When that happens, you have to resolve these conflicts and not push the conflicting changes.

I would really like to help you with this sometime tomorrow because it seems that you have a hard time with fixing everything :)

**Comment by PouriyaHos** [comment]
No, because me and Ahmad did it together, in my system didn't have strange characters

**Comment by AhmadAlkaati** [comment]
> No, because me and Ahmad did it together, in my system didn't have strange characters
I agree with @josipgrgic about having a zoom call with him and sharing your screen so he can see what needs to be fixed now.
Yesterday I managed that we push your latest code to the same branch where you had your code last time but I am not sure how to solve conflicts.
Surely Josip knows a lot more about github :)

**Comment by josipgrgic** [comment]
> > No, because me and Ahmad did it together, in my system didn't have strange characters
> > I agree with @josipgrgic about having a zoom call with him and sharing your screen so he can see what needs to be fixed now.
> > Yesterday I managed that we push your latest code to the same branch where you had your code last time but I am not sure how to solve conflicts.
> > Surely Josip knows a lot more about github :)

The issue is most surely because you didn't merge the main branch into this one in a while. We can make a call, but I was and still am busy today, so maybe tomorrow before or after the wrap-up meeting?

**Pull request closed without merging**